### ISSUE TITLE
Model containing "\\" causes parsing to fail

### ISSUE THREAD

**Issue by MartinSStewart** [layflags/elm-monitor#9]
A model containing an escaped backslash will cause elm-monitors parsing to fail.

Here's a simple example
```
main : Program () String ()
main =
    Monitor.sandbox
        { init = "\\"
        , view = \_ -> Html.button [ Html.Events.onClick () ] [ Html.text "produces an error in console" ]
        , update = \_ model -> model
        }
```

This occurred in version 0.0.9

**Comment by ChristophP** [comment]
Published as `0.010`

**Comment by layflags** [comment]
Thx for reporting @MartinSStewart 👍
Thx for fixing @ChristophP 👍

**Comment by MartinSStewart** [comment]
This fix seems to have introduced a new error. The example program now produces this error
![image](https://user-images.githubusercontent.com/5068391/56847589-36ebe080-68dd-11e9-9992-f11828a52f8d.png)
It produces this error even if I change the `"\\"` to something else.

**Comment by ChristophP** [comment]
@MartinSStewart Hmm, I wonder how The changes could cause this runtime error. Maybe a bug in the compiled Elm code? I can take a look but I wonder if this has bigger implications on the Elm side.

**Comment by MartinSStewart** [comment]
The bug maybe have been present before but wasn't visible until the previous bug was fixed.  I did consider that it might have been something wrong elsewhere but it goes away if I switch from `Monitor.element` to `Browser.element`.

**Comment by ChristophP** [comment]
@MartinSStewart Yes that makes me think if its something unrelated to the fix I made. I wonder if its just the fact that your model is a plain string (which maybe nobody has tested before). Could you verify that by downgrading to the previous version and check if your example also throws that error when using a string that doesn't contain slashes.

**Comment by ChristophP** [comment]
I don't have a computer right now

**Comment by MartinSStewart** [comment]
This bug is present in 0.0.9 as well.

**Comment by ChristophP** [comment]
As I suspected. Thanks for checking. Must be something not related to parsing then. @layflags Can you check out the problem that plain string models will throw an error. Maybe that should be a new issue as well.

**Comment by MartinSStewart** [comment]
I think it has to do with msg being the unit type.
```
type alias Model_ =
    { array : Array Int
    }


type Msg_
    = Blah


main : Program () Model_ Msg_
main =
    Monitor.sandbox
        { init = { array = Array.fromList [ 5 ] }
        , view = \_ -> Html.button [ Html.Events.onClick Blah ] [ Html.text "produces an error in console" ]
        , update = \_ model -> model
        }
```
This code works fine, but if I replace `Msg_` with `()` then I get the `n is null` exception.

**Comment by ChristophP** [comment]
Yeah, the unit type maps to null. But I am not sure why that causes the error. I am sure @layflags can say more about that.

**Comment by layflags** [comment]
I reopen the issue because of the other bug

**Comment by ChristophP** [comment]
@layflags lets rather open a New one. The problems are unrelated

**Comment by layflags** [comment]
Closed in favour of #13
